connectors/PartnerCenterReferrals/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1097 @@
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Microsoft.Partner.Service.PartnerReferrals.Contracts.V3.Referral"
+            }
+          },
+          "400": {
+            "description": "There was a missing or an invalid input. The response body will provide the error details."
+          },
+          "401": {
+            "description": "The request was not authenticated. The client needs to authenticate with the partner API service first."
+          },
+          "403": {
+            "description": "The request was authenticated but was refused since the caller does not have the rights to invoke it."
+          },
+          "404": {
+            "description": "The resource is not found."
+          },
+          "500": {
+            "description": "The partner API service or one of its dependencies failed to fulfill the request. Callers may retry this."
+          }
+        },
+        "summary": "Get a referral by Id",
+        "tags": [
+          "Referrals"
+        ]
+      },
+      "put": {
+        "consumes": [
+          "application/json"
+        ],
+        "deprecated": false,
+        "description": "Updates the referral matching the given referral identifier",
+        "operationId": "UpdateReferralById",
+        "parameters": [
+          {
+            "description": "The unique id for this referral.",
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Identifier of the referral."
+          },
+          {
+            "in": "body",
+            "name": "odataReferral",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Microsoft.Partner.Service.PartnerReferrals.Contracts.V3.Referral"
+            }
+          },
+          {
+            "description": "Used for if match header from the client.",
+            "in": "header",
+            "name": "if-match",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Used for if match header from the client"
+          },
+          {
+            "default": "application/json",
+            "description": "Content type of the request",
+            "in": "header",
+            "name": "content-type",
+            "required": true,
+            "type": "string",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "description": "The unique identifier to track the request",
+            "in": "header",
+            "name": "MS-CorrelationId",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Correlation Identifier",
+            "x-ms-visibility": "important"
+          },
+          {
+            "default": "60858c1b-2062-40ad-b42c-996b6b5b047d",
+            "description": "Optional, ID to identify the caller - PowerAutomate",
```

Comment:
@didalal Please add a space between Power and Automate like below:

`"description": "Optional, ID to identify the caller - Power Automate",`